test/Microsoft.AspNetCore.SignalR.Client.FunctionalTests/HubConnectionTests.cs

```diff
@@ -864,15 +864,14 @@ bool ExpectedErrors(WriteContext writeContext)
             }
         }
 
-        [Theory]
-        [MemberData(nameof(TransportTypes))]
-        public async Task CheckHttpConnectionFeatures(HttpTransportType transportType)
+        [Fact]
+        public async Task CheckHttpConnectionFeatures()
```

Comment:
I didn't want the test I was adding (below) to vary by transport because it seemed unnecessary. This was another test that was very similar (not testing anything transport-specific) so I just removed the parameter from it as well. If people things we should revert this, I should make my test vary by transport too.